MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,45 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+#define DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+
+#include <Eigen/Core>
+#include "MagneticField/ParametrizedEngine/interface/ParabolicParametrizedMagneticField.h"
+
+using Vector3f = Eigen::Matrix<float, 3, 1>;
```

Comment:
`Vector3f` is not used in the code, please remove it.

Reply:
Removed.